### DISCUSSION TITLE
some height classes like .h-1/2 etc are not working...?

### DISCUSSION THREAD

**Discussion by Ochirpurev20** [tailwindlabs/tailwindcss#3022] [Help] [answered]
Here is my code that height classes are not working: 
:::::::::::
<body class="bg-white text-base">
    
    <section>
        <div class="h-10 text-white px-10 py-2">
            <strong class=" text-base"> mail@pairlink.mn</strong>
        </div>
    
    </section>
    
    <div class=" bg-blue-300 **h-2/3**">

        <div class="container">
        <p class="text-base text-black">PAIRLINK LLC main </p>        
        </div>
    </div>
    
    <section class=" bg-green-400 **h-1/2**"> <p class="text-white"> Solutions we provide</p></section>
    <section> </section>
    <section> </section>
    
    <section> </section>
    
    
</body>

**Comment by simonswiss** [comment, marked as answer]
Hey!

These will only work relatively to the parent element from where the class is applied, since they're percentage values.

I have made an example where your code is wrapped in a `h-screen` container (full height of the viewport). You can see the two sections taking `2/3` and `1/2` of the height in that case:

https://play.tailwindcss.com/HRNm5g8JF7

Same deal if the parent is `h-80` (20rem) in height - the two sections will use `2/3` and `1/2` of that height:

https://play.tailwindcss.com/pYMkx7taUl

Hope it helps!

**Comment by Ochirpurev20** [comment]
thank you very much.

that helped a lot.
and it was my bad.

**Reply by simonswiss** [reply]
Glad it was helpful! 👍
